bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -208,9 +214,13 @@
    static final boolean STATE_IS_SHOW_IN_CHART_TOOLBAR_RUN_DYN_VERTICAL_RATIO_DEFAULT        = false;
    static final boolean STATE_IS_SHOW_IN_CHART_TOOLBAR_SWIM_STROKES_DEFAULT                  = true;
    static final boolean STATE_IS_SHOW_IN_CHART_TOOLBAR_SWIM_SWOLF_DEFAULT                    = false;
-
+   //CUSTOM TRACKS
+   static final boolean STATE_IS_SHOW_IN_CHART_TOOLBAR_CUSTOM_TRACKS_DEFAULT                 = false;
+   //
    private static final String GRAPH_CONTRIBUTION_ID_SLIDEOUT                       = "GRAPH_CONTRIBUTION_ID_SLIDEOUT";                      //$NON-NLS-1$
 
+   //CUSTOM TRACKS
+   private static final String GRAPH_CONTRIBUTION_ID_CUSTOM_TRACKS                  = "GRAPH_CONTRIBUTION_ID_CUSTOM_TRACKS";                 //$NON-NLS-1$
```

Comment:
the "//CUSTOM TRACKS" was there for me to quickly find back where I had modified the code, my plan was to remove all of those comments at the end once everything is ready.
So I'll remove those comments now since it is messing up things for you.